src/libsystemd/sd-login/sd-login.c

```diff
@@ -503,6 +503,33 @@ _public_ int sd_uid_get_display(uid_t uid, char **session) {
         return 0;
 }
 
+_public_ int sd_uid_get_login_time(uid_t uid, uint64_t *usec) {
+        _cleanup_free_ char *p = NULL, *s = NULL;
+        usec_t t;
+        int r;
+
+        assert_return(usec, -EINVAL);
+
+        r = file_of_uid(uid, &p);
+        if (r < 0)
+                return r;
+
+        r = parse_env_file(NULL, p, "REALTIME", &s);
```

Comment:
Hmm, I think this would get the start time of lingering users too, which don't have active sessions and can't be called _logged in_ 🤔

Maybe get the state first, check if it's a lingering user, and return an error if that's the case?

Reply:
Yes, I think we should check if the state is the one of an active user.
Need to find out how to do that now...